src/vsr/message_header.zig

```diff
@@ -635,26 +667,33 @@ pub const Header = extern struct {
         reserved_frame: [16]u8 = [_]u8{0} ** 16,
 
         checkpoint_id: u128,
+        checkpoint_id_reserved: u128 = 0,
         /// The corresponding request message's checksum.
-        parent: u128,
-        client: u128,
+        /// TODO: Add this back and remove checkpoint_id instead, once checkpoint_id is added to
+        /// the Prepare header.
+        // parent: u128,
+        // parent_reserved: u128 = 0,
```

Comment:
I'm just commenting these out since I expect to add them all back in my next PR.

(Swapping `checkpoint_id` over to `Prepare` is a more extensive change than simply temporarily disabling `parent` verification.)